doc/data/messages/u/useless-option-value/bad.py

```diff
@@ -0,0 +1,3 @@
+"""'bad-continuation' was removed from pylint in https://github.com/PyCQA/pylint/pull/3571"""
```

Comment:
The empty `good.py` looks a bit weird:
https://pylint--6824.org.readthedocs.build/en/6824/user_guide/messages/information/useless-option-value.html

Reply:
Does https://pylint--6824.org.readthedocs.build/en/6824/user_guide/messages/refactor/useless-option-value.html looks better ?

Reply:
Yes I think so!